Formula/watch.rb

```diff
@@ -25,9 +26,11 @@ class Watch < Formula
 
   def install
     system "autoreconf", "-fiv"
+    system "echo", "\"%s/\\\\(CURSES_LIB=ncurses\\\\)w/\\\\1/\\nwq\"", "|", "ed -s configure"
```

Comment:
Why do we need to hack the configure script?

Reply:
@ThoFrank Why do you use `ed`, when you can `inreplace` without spawning another process? Not to mention, you're trying to solve the problem in a very strange way.

Reply:
Yeah I don't really know what I am doing there. Sorry for the mess.
The problem is that building watch with the "--enable-watch8bit" flag requires ncursesw. It exists on my machine, but is only a link to ncurses.
I found the hack googling around, but if you have an idea how to deal with the problem properly please feel free to help.

Reply:
@ThoFrank I will look into the matter today. As I remember, `watch` asked for `ncurses/ncursesw` only for `--enable-watch8bit`, but I don't think it really does matter if it 'classic' or 'narrow'.